sentry_redis_tools/retrying_cluster.py

```diff
@@ -22,19 +16,13 @@ class RetryingRedisCluster(RedisCluster):  # type: ignore
 
     def execute_command(self, *args: Any, **kwargs: Any) -> Any:
         try:
-            return super(self.__class__, self).execute_command(*args, **kwargs)
+            return super(self.__class__, self).execute_command(*args, **kwargs)  # type: ignore[no-untyped-call]
```

Comment:
this is some weird python2ism -- this can be `return super().execute_command(*args, **kwargs)` -- might also fix the type ignore

Reply:
For education purposes: If that works, which is really great, why did we ever need `self.__class__` in the first place?

Reply:
python 2 :) -- and technically as written it's incorrect / imprecise for multiple inheritance

0-arg super is a new python 3 convenience

Reply:
Nice. Thanks!